src/v/net/connection_rate.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "seastarx.h"
+
+#include <seastar/core/gate.hh>
+#include <seastar/core/lowres_clock.hh>
+#include <seastar/core/semaphore.hh>
+#include <seastar/core/sstring.hh>
+#include <seastar/net/net.hh>
+
+#include <absl/container/flat_hash_map.h>
+#include <absl/container/node_hash_map.h>
+
+#include <chrono>
+
+using namespace std::chrono_literals;
```

Comment:
make sure to clean-up your headers. I don't think most of these are used

Reply:
EDIT: I see a bunch of stuff is added to this header in the next commit so no big deal.